### PR TITLE
:chart_with_downwards_trend: Setup monitoring for stage systems

### PR DESCRIPTION
**Pull request by harshad16**
Setup monitoring for stage systems
Signed-off-by: Harshad Reddy Nalla <hnalla@redhat.com>

## Related Issues and Dependencies

Related-to: https://github.com/thoth-station/thoth-application/issues/2455

## Description

All the monitoring components are deployed behind the VPN.

Prometheus: http://prometheus-aicoe-infra-prod.apps.ocp4.prod.psi.redhat.com
Pushgateway: http://pushgateway-aicoe-infra-prod.apps.ocp4.prod.psi.redhat.com
Grafana: https://grafana-aicoe-infra-prod.apps.ocp4.prod.psi.redhat.com

### CODE REVIEW

File: monitoring/overlays/ocp4-stage/prometheus/prometheus.yaml [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+        app: prometheus
+      name: prometheus
+    spec:
+      serviceAccountName: prometheus
+      containers:
+        - name: prometheus
+          args:
+            - --storage.tsdb.retention=7d
+            - --storage.tsdb.min-block-duration=2h
+            - --storage.tsdb.max-block-duration=2h
```
Comment (VannTen) [inline monitoring/overlays/ocp4-stage/prometheus/prometheus.yaml:27]: Isn't this prometheus default ?

**Review by VannTen — COMMENTED** [review]
Looks sane to me
/lgtm

File: monitoring/overlays/ocp4-stage/prometheus/prometheus.yaml (hunk 1 above)
Comment (harshad16) [reply monitoring/overlays/ocp4-stage/prometheus/prometheus.yaml:27]: yes, this is the default, I got the value here, so if the workload causes an issue, we can update it.

**Comment by harshad16** [comment]
Thanks for the review 
/approve

**Pull request merged**